addon/components/nrg-new-features/component.js

```diff
@@ -0,0 +1,43 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { alias, bool } from '@ember/object/computed';
+import { task } from 'ember-concurrency';
+import layout from './template';
+
+export default Component.extend({
+  layout,
+
+  whatsNewService: service('whats-new'),
+
+  isOpen: bool('content'),
+
+  headerText: alias('whatsNewService.headerText'),
+
+  primaryButtonText: alias('whatsNewService.primaryButtonText'),
+  secondaryButtonText: alias('whatsNewService.secondaryButtonText'),
+
+  checkContent: task(function*() {
+    let content = yield this.get('whatsNewService.getContent').perform();
+    if (content) {
```

Comment:
Since the `isOpen` property is tied to a bool, we shouldn't need to check for falsy values before setting the content variable.